src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cleared = sourceString.split(';');
+
+  return cleared.reduce((accum, style) => {
+    const n = style
+      .split(':')
+
+      .map((element) => element.trim());
+
+    const [key, value] = n;
+
+    accum[key] = value;
+
+    return accum;
+  }, {});
```

Comment:
[CODE KNOWLEDGE] - There is no need to split the string into an array and then reduce it into an object. You can use the 'map' method to transform each style declaration into an object and then use 'Object.assign' to combine them into a single object.